perf/istio/gateway-bouncer/templates/fortio-server.yaml

```diff
@@ -0,0 +1,59 @@
+# A simple 'Hello World' style service running behind the IngressGateway that
+# is configured using standard K8S Ingress with a custom Ingress class defined
+# in Pilot mesh config.
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fortio-server
+spec:
+  ports:
+  - port: 8080
+    name: http-echo
+  - port: 8079
+    name: grpc-ping
+  selector:
+    app: fortio-server
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: fortio-server
+spec:
+  template:
+    metadata:
+      labels:
+        app: fortio-server
+        version: v1
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      containers:
+      - name: fortio-server
+        image: {{ .Values.fortioImage }}
+        imagePullPolicy: Always
+        ports:
+         - containerPort: 8080
+         - containerPort: 8079
+        args:
+          - server
+        resources:
+          requests:
+            cpu: 1000m
```

Comment:
These seem a bit high, does fortio need this much?

Reply:
It's convenient to have at least one core, so I can easily dial in more load by increasing the number of client replicas if we're interested in what is happening under the load. I only went with one core though (allconfig setup is using like 4x cores for the Fortio Server -- I don't think we need that much for this particular scenario though).

Reply:
We can probably keep it at 0.2 for 200 qps.  Usually it's 3000 qps per core for sidecar, fortio can handle much more.